openwisp_controller/connection/models.py

```diff
@@ -3,9 +3,10 @@
 
 from django.core.exceptions import ValidationError
 from django.db import models
-from django.utils import timezone
+from django.dispatch import Signal
 from django.utils.functional import cached_property
 from django.utils.module_loading import import_string
+from django.utils.timezone import now
```

Comment:
why this change? Avoid unnecessary changes as much as you can.
Keep in mind: http://openwisp.io/docs/developer/contributing.html#avoiding-unnecessary-changes